### PR TITLE
Update harvard-university-for-the-creative-arts.csl

### PR DESCRIPTION
**Pull request by ibadger**
Add variable Director, previously had used only Author but this wasn't displaying the Director in references produced in Zotero. If there is a better way to do it do let me know

### CODE REVIEW

File: harvard-university-for-the-creative-arts.csl [hunk 1]
```diff
@@ -278,7 +278,7 @@
             </group>
           </if>
         </choose>
-        <names variable="author" prefix="Directed by ">
+        <names variable="author director" delimiter=" " prefix="Directed by ">
```
Comment (bwiernik) [inline harvard-university-for-the-creative-arts.csl:281]:
```suggestion
        <names variable="director">
          <label variable="director" text-case="capitalize-first" suffix=" "/>
```

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks, I updated to use a more robust approach.

**Comment by ibadger** [comment]
I added a Macro for Director and resubmitted, it seemed to validate but am unsure if it is the best way to go about it. Sorry for the delay I have been away from work, and am a bit slow when it comes to coding

**Comment by ibadger** [comment]
I made some changes a while ago, but forgot about it  - does it work now?

**Pull request closed without merging**